src/app.html

```diff
@@ -17,10 +17,10 @@
     <a class="nav-link" id="nav-about" href="#/about">About</a>
     <a class="nav-link open" id="nav-header" href="#/header">Header</a>
     <a class="nav-link" id="nav-mime" href="#/mime">MIME types</a>
-    <a class="nav-link" id="nav-security" href="#/global-security">Security</a>
+    <a class="nav-link" id="nav-global-security" href="#/global-security">Security</a>
```

Comment:
Since there are so many links here and they have different lengths, lets nest the link text on their own line. That way, all link text will be aligned to the same column, so we can vertically scan the code to know what's what.